### ISSUE TITLE
change to auth user and pwd jupyter

### ISSUE THREAD

**Issue by johnfelipe** [TrivadisPF/platys-modern-data-platform#547]
```
(base) jovyan@jupyter:~$ sudo tljh-config set auth.type nativeauthenticator.NativeAuthenticator
[sudo] password for jovyan:
Sorry, try again.
[sudo] password for jovyan:
sudo: no password was provided
sudo: 1 incorrect password attempt
(base) jovyan@jupyter:~$ sudo su
[sudo] password for jovyan:
sudo: no password was provided
(base) jovyan@jupyter:~$ tljh-config set auth.type nativeauthenticator.NativeAuthenticator
bash: tljh-config: command not found

```

https://tljh.jupyter.org/en/latest/howto/auth/nativeauth.html

wich credentials for jovyan?

**Comment by gschmutz** [comment]
@johnfelipe the link you point to to is for jupyterhub, isn't it?

**Comment by gschmutz** [comment]
@johnfelipe the password for jupyter is currently hardcoded in the docker-compose.yml. Will see if I can support all passwords through the use of docker `secrets`. By that it could be changed easily and outside of the `docker-compose.yml`.

**Comment by johnfelipe** [comment]
> @johnfelipe the link you point to to is for jupyterhub, isn't it?

Is possible add like a features Jupyter hub, I add like issue to...

**Comment by johnfelipe** [comment]
> @johnfelipe the link you point to to is for jupyterhub, isn't it?

Yes is for jupyter hub, then is not works here because this is notebook

**Comment by johnfelipe** [comment]
ping me pls when HUBS is ready to use

**Comment by gschmutz** [comment]
Support for JupyterHub has been updated and will be available once the 1.16.0 release is out. Closing the issue.
